### PR TITLE
Update spaceapi server to 0.6

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

**Comment by rnestler** [comment]
@dbrgn This is waiting for https://github.com/spaceapi-community/spaceapi-server-rs/pull/107 to be merged.

**Comment by rnestler** [comment]
@dbrgn Now it is ready for review :slightly_smiling_face: Finally the hackerspace with 2 members in the spaceapi/core team also supports v14 :wink:

**Comment by rnestler** [comment]
@dbrgn We don't have reviews enabled on this repo as it seems. Should we?

File: src/main.rs [hunk 1]
```diff
@@ -76,12 +76,14 @@ fn main() {
         .add_project("https://github.com/coredump-ch/")
         .add_cam("https://webcam.coredump.ch/cams/ultimaker_0.jpg")
         .add_extension("ccc", "chaostreff")
+        .state(api::State {
+            open: Some(false),
```
Comment (dbrgn) [inline src/main.rs:80]: Shouldn't this be `None` by default?

Comment (rnestler) [reply src/main.rs:80]: Debatable. But if you look at the message it is the closed message.

Comment (rnestler) [reply src/main.rs:80]: But you are right: The `StateFromPeopleNowPresent` will set it to `true` / `false` depending on the sensor value or leave it `None` if no sensor is available, which is nice.

Comment (rnestler) [reply src/main.rs]: Fixed.

**Review by dbrgn — APPROVED** [review]

**Pull request merged**